webrtc/src/main/java/com/baeldung/webrtc/WebRTCDemoApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.baeldung.webrtc;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class WebRTCDemoApplication {
+
+	public static void main(String[] args) {
```

Comment:
Use spaces instead of tabs for indentation.

Reply:
Have made this change.